### ISSUE TITLE
Crash on 1.12.2 version

### ISSUE THREAD

**Issue by Jospuntnl** [Electrical-Age/ElectricalAge#950]
After compiling the 1.12.2 version the game will crash on trying to run it. This is the crash report:
https://pastebin.com/SAGLzzyB
Can somebody tell me what's going wrong here?

**Comment by jrddunbr** [comment]
Simple. Electrical Age doesn't support 1.12.

**Comment by misterB0420** [comment]
to elaborate further, for the OP and anyone else googling this issue, 1,12 is a version number for the mod itself, not the version of minecraft it's for. the newest version, 1.14.2 is for minecraft 1.7.10. though I've heard they're still doing work on this, i think it's become more of a pet project, and I don't really expect any more releases. A shame, this was one of, if not the best mod I'd ever seen. Don't feel bad, it's an easy mistake to make, I've been with the modding community since it's birth ten or so years ago, and I still make this mistake once in a while. Sometimes it's easy to miss one sentence in one small paragraph that clarifies that, especially if you've been building a modpack for yourself for hours and you're really tired.

**Comment by jrddunbr** [comment]
MC 1.12 porting is only taking so long because we need developers to step up to the plate.

Currently, I'm the project leader on Eln2 which will be the complete rewrite of code to MC 1.12. I would anticipate that at the current rate of progress that porting will take a year or more.

If you know competent EE's who know some Java or Kotlin, that would be awesome.

**Comment by Bricklou** [comment]
Is the repo of Eln2 is public ? I want to see if I can help for something.

**Comment by jrddunbr** [comment]
https://github.com/eln2 and https://eln2.org/ but because of the virus we are really behind on that summer 2020 expectation.

**Comment by jrddunbr** [comment]
This week's project is to refactor the UMC Gradle project to the new codebase, and to get the `core` mna working with integration.

**Comment by jrddunbr** [comment]
Oh, right, the website is super outdated because I couldn't get it to build locally. Read eln2-web repo docs folder for website content.

**Comment by Bricklou** [comment]
I have some specialization for web fullstack development so if you want some help for a website I can do it.

**Comment by jrddunbr** [comment]
Nah, I just need to fix my pip install and build the website again I think. Nothing too difficult. The website is statically hosted on GitHub pages so there isn't much of a backed honestly

**Comment by plajdo** [comment]
@jrddunbr hey. If you need any help with development I think I maybe could help. I don't really have too much experience with Minecraft modding but I have about 7 years of general Java knowledge and I work in Kotlin past 2 years. I also got a bit more time than I usually have due to the virus so if you want hit me up somewhere and we'll see if I could help in any way.

**Comment by jrddunbr** [comment]
Are you in the Electrical Age Discord? That's where we do most devchat.

**Comment by jrddunbr** [comment]
https://discord.gg/YjK2JAD
